rcl/test/rcl/test_init.cpp

```diff
@@ -271,3 +280,21 @@ TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_get_instance_id)
   ret = rcl_context_fini(&context);
   EXPECT_EQ(ret, RCL_RET_OK);
 }
+
+TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_init_options_access) {
+  rcl_init_options_t init_options = rcl_get_zero_initialized_init_options();
+  rcl_ret_t ret = rcl_init_options_init(&init_options, rcl_get_default_allocator());
+  ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+
+  rmw_init_options_t * options = rcl_init_options_get_rmw_init_options(&init_options);
+  ASSERT_NE(nullptr, options);
+  EXPECT_EQ(0u, options->instance_id);
+  EXPECT_EQ(nullptr, options->impl);
+
+  const rcl_allocator_t * options_allocator = rcl_init_options_get_allocator(&init_options);
+  EXPECT_TRUE(rcutils_allocator_is_valid(options_allocator));
+
+  rcl_init_options_t init_options_dst = rcl_get_zero_initialized_init_options();
+  EXPECT_EQ(RCL_RET_OK, rcl_init_options_copy(&init_options, &init_options_dst));
```

Comment:
@Blast545 same about an `rcl_init_options_fini()` call missing, this time for `init_options_dst`.